### PR TITLE
SILKIT-1525: Use bulk-update endpoint of dashboard backend service

### PR DESCRIPTION
**Pull request by VDanielEdwards**
## Subject

These changes allow the dashboard client in SIL Kit to use the bulk-update endpoint (currently only available in a feature branch of the dashboard that is not publically available).

It allows for much larger simulations, as multiple changes can be combined into a single HTTP request.

## Developer checklist (address before review)

- [x] Changelog.md updated
- [ ] Prepared update for depending repositories
- [x] Documentation updated (public API changes only)
- [ ] API docstrings updated (public API changes only)
- [x] Squash and merge &rarr; proper PR title

### CODE REVIEW

**Comment by VDanielEdwards** [comment]
NOTE: The branch was rebased onto the (soon-to-be-merged) branch `dev_disable_broken_ubuntu1804_uploads` from [PR-73](https://github.com/vectorgrp/sil-kit/pull/73). Therefore, there will be a final rebase onto main after that one was merged.

**Review by NadegeNag — COMMENTED** [review]
Hab ich richtig gesehen, dass die Optimierung nur über  ExperimentalEnableDashboardBulkUpdates in der Konfigdatei aktivierbar ist
und über CLI immer aus ist? 
Dann dokumentiere ich es so im Dashboard README. :)

**Comment by VDanielEdwards** [comment]
> Hab ich richtig gesehen, dass die Optimierung nur über ExperimentalEnableDashboardBulkUpdates in der Konfigdatei aktivierbar ist und über CLI immer aus ist? Dann dokumentiere ich es so im Dashboard README. :)

I'm likely going to get some feedback from stakeholders today, so it's probably best if you wait until afternoon 👍

**Comment by VDanielEdwards** [comment]
@VDanielEdwards Note-to-Self: Check where the registry configuration schema is located and maintained (it is available in the schema store, but not in the `sil-kit` repository ?).

**Review by KonradBkd — APPROVED** [review]
Looked hard, couldn't really find something to nag about. Feel free to request another review if the changes of the dashboard version check are in.

**Comment by VDanielEdwards** [comment]
@KonradBkd we replaced the flag with auto-detection of the bulk-update availability. The code has been streamlined slightly.

File: SilKit/source/dashboard/CreateDashboardInstance.hpp [hunk 1]
```diff
@@ -13,6 +13,7 @@
 namespace VSilKit {
 
 
+// NB: After some time in the wild, the bulk update will become the default and the parameter can be removed.
```
Comment (KonradBkd) [inline SilKit/source/dashboard/CreateDashboardInstance.hpp:16]: Comment no longer valid

**Review by KonradBkd — APPROVED** [review]
LGTM (except the obsolete comment)

**Pull request merged**